bundles/org.eclipse.ui.workbench/Eclipse UI/org/eclipse/ui/internal/WorkbenchPlugin.java

```diff
@@ -210,6 +217,7 @@ public class WorkbenchPlugin extends AbstractUIPlugin {
 	public WorkbenchPlugin() {
 		super();
 		inst = this;
+		recentNewWizardsPreferenceManager = new RecentNewWizardsPreferenceManager();
```

Comment:
Never ever do anything in bundle constructor! Any bundle initialization code should be added in the `start()` method.